sdk/database/newdbplugin/grpc_server_test.go

```diff
@@ -137,72 +129,28 @@ func TestCleanNumbers(t *testing.T) {
 				"foo": 42,
 			},
 		},
-		"json.Number integer": {
-			input: map[string]interface{}{
-				"foo": json.Number("42"),
-			},
-			expected: map[string]interface{}{
-				"foo": int64(42),
-			},
-		},
-		"json.Number float": {
-			input: map[string]interface{}{
-				"foo": json.Number("42.123"),
-			},
-			expected: map[string]interface{}{
-				"foo": float64(42.123),
-			},
-		},
-		"bad json.Number": {
-			input: map[string]interface{}{
-				"foo": json.Number("bar"),
-			},
-			expected: map[string]interface{}{
-				"foo": json.Number("bar"),
-			},
-		},
-		"recursive integer": {
-			input: map[string]interface{}{
-				"foo": map[string]interface{}{
-					"bar": json.Number("42"),
-				},
-			},
-			expected: map[string]interface{}{
-				"foo": map[string]interface{}{
-					"bar": int64(42),
-				},
-			},
-		},
-		"recursive float": {
+		"floats ": {
 			input: map[string]interface{}{
-				"foo": map[string]interface{}{
-					"bar": json.Number("42.123"),
-				},
+				"foo": 42.2,
 			},
 			expected: map[string]interface{}{
-				"foo": map[string]interface{}{
-					"bar": float64(42.123),
-				},
+				"foo": 42.2,
 			},
 		},
-		"recursive no numbers": {
```

Comment:
Should we keep non-numeric test cases to make sure that conversion only happens when appropriate?

Reply:
Nevermind, didn't realize I was looking at a subset of changes.